code/core/src/manager/components/layout/LayoutProvider.tsx

```
@@ -26,11 +26,15 @@ const LayoutContext = createContext<LayoutContextType>({
 
-export const LayoutProvider: FC<PropsWithChildren> = ({ children }) => {
+export const LayoutProvider: FC<
+  PropsWithChildren & {
+    forceDesktop?: boolean;
+  }
+> = ({ children, forceDesktop }) => {
   const [isMobileMenuOpen, setMobileMenuOpen] = useState(false);
   const [isMobileAboutOpen, setMobileAboutOpen] = useState(false);
   const [isMobilePanelOpen, setMobilePanelOpen] = useState(false);
-  const isDesktop = useMediaQuery(`(min-width: ${BREAKPOINT}px)`);
+  const isDesktop = forceDesktop ?? useMediaQuery(`(min-width: ${BREAKPOINT}px)`);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)